config.json

```diff
@@ -218,6 +218,10 @@
      ]
   }, {
      "difficulty": 1,
+     "slug": "triangle",
+     "topics": []
```

Comment:
At a minimum, topics needs to include `control flow (if-else statements)`. There may be others that apply as well, see the list https://github.com/exercism/x-common/blob/master/TOPICS.txt